sigma/cli/convert.py

```diff
@@ -110,6 +108,12 @@ def fail(self, message: str, param, ctx):
     "-c",
     help="Select method for generation of correlation queries. If not given the default method of the backend is used."
 )
+@click.option(
```

Comment:
Do we really need another option? Filters are "just" another type of input file that could be passed on the command line as argument.

Reply:
Technically yes, if you go down it just appends the rules together but from a purely end-user experience, it makes sense to supply it as an argument. 

It comes from how I expect every input to map to some sort of output (rules, correlation rules) but technically filters don't produce an output.